index.html

```diff
@@ -0,0 +1,244 @@
+                        </li>
+                        <li>
+                            <a href="#">Mypage</a>
+                        </li>
+                        <li>
+                            <a href="#">Logout</a>
+                        </li>
+
+                    </ul>
+
+                </div>
+                <div class="menu-desktop">
+                    <div class="menu-atlas">
+                        <img src="./img/logo.png" alt="Menu Icon">
+
+                    </div>
+                    <ul class="menu-ul">
+                        <li class="menu-li">
+                            <a href="./about.html">About</a>
+                        </li>
+                        <li class="menu-li">Tickets</li>
+                        <li class="menu-li">Join</li>
+                        <li class="menu-li">Sponsor</li>
+                        <li class="menu-li">News</li>
+                        <li class="menu-li cc">CC Campaign</li>
+
+                    </ul>
+                </div>
+
+            </nav>
+        </div>
+
+    </header>
+    <main>
+        <section>
+            <div class="welcome">
+                <div class="intro">
+                    <h3> "Hello! Sharing World"</h3>
+                    <h1> "CREATIVE COMMONS <br> GLOBAL SUMMIT <br> 2015"</h1>
+                    <p>Praesent porttitor, mi quis pulvinar rutrum, nunc dolor rutrum enim, eu tempor nulla felis eu
+                        orci.
+                        Ut eu turpis nisi. Pellentesque convallis, purus vitae mollis ultrices, ex lectus consectetur
+                        nulla,
+                        et blandit ipsum nibh sit amet ipsum.</p>
+                    <h2> 2015.10.15(THU) ~ 16(FRI) <BR> @ National Museum of Korea, Art Center Nabi and More </h2>
+
+
+
+                </div>
+            </div>
+        </section>
+    </main>
+
+
+    <section id="about">
+        <div class="lets-grow">
+            <h3 class="lets-grow-title">Main Program</h3>
+            <ul class="programs">
+                <li class="Lecture">
+                    <img src="./img/Tools.png" alt="Tools">
+                    <h4>Lecture</h4>
+                    <p>Lorem ipsum dolor, sit amet consectetur adipisicing elit.</p>
+
+                </li>
+                <li class="cc-Exhibition">
+                    <img src="./img/Bulb.png" alt="Bulb">
+                    <h4>CC Exhibtion</h4>
+                    <p>Lorem ipsum dolor sit amet.</p>
+
+                </li>
+                <li class="forum">
+                    <img src="./img/Handshake.png" alt="Forum">
+                    <h4>Forum</h4>
+                    <p>Lorem ipsum dolor sit amet consectetur adipisicing elit.</p>
+
+                </li>
+                <li class="workshop">
+                    <img src="./img/Medal.png" alt="Medal">
+                    <h4>Workshop</h4>
+                    <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. </p>
```

Comment:
Great job @Mutai-Gilbert 👍. Just as the previous reviewer mentioned, kindly use the correct spaces between the boxes.
I think in order to match the requirement, you can reduce the gap in between the boxes.😊
| Figma Design |Your Design |
| :---:   | :---: | 
| <img width="770" alt="Screenshot 2022-11-24 at 20 49 47" src="https://user-images.githubusercontent.com/73854999/203857316-905d493e-514a-443f-bffa-411f47cc04fd.png">|<img width="1024" alt="Screenshot 2022-11-24 at 20 50 17" src="https://user-images.githubusercontent.com/73854999/203857342-186816ba-3222-4dc2-90d9-2baf8297793a.png">|